cmd/nsp/main.go

```diff
@@ -93,6 +87,8 @@ func main() {
 	))
 	nspAPI.RegisterTargetRegistryServer(server, targetRegistryServer)
 	nspAPI.RegisterConfigurationManagerServer(server, configurationManagerServer)
+	healthServer := grpcHealth.NewServer()
+	grpc_health_v1.RegisterHealthServer(server, healthServer)
```

Comment:
Would it still be possible to use health.SetServingStatus with this way to create the health server? The NSP is watching the configmap using the Kubernetes API, so it would be good to have it part of the probes.